index.html

```diff
@@ -5,29 +5,49 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="style.css">
+  <script src="script.js" defer></script>
   <title>Benjamin Portfolio</title>
 </head>
 <body>
+  <section id="menu-wrapper" class="display-none">
+    <div class="menu-layer container">
+      <button type='button'>
+        <svg id="menu-close" width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg">
+          <path fill-rule="evenodd" clip-rule="evenodd"
+            d="M0.292893 0.292893C0.683417 -0.0976311 1.31658 -0.0976311 1.70711 0.292893L7 5.58579L12.2929 0.292893C12.6834 -0.0976311 13.3166 -0.0976311 13.7071 0.292893C14.0976 0.683417 14.0976 1.31658 13.7071 1.70711L8.41421 7L13.7071 12.2929C14.0976 12.6834 14.0976 13.3166 13.7071 13.7071C13.3166 14.0976 12.6834 14.0976 12.2929 13.7071L7 8.41421L1.70711 13.7071C1.31658 14.0976 0.683417 14.0976 0.292893 13.7071C-0.0976311 13.3166 -0.0976311 12.6834 0.292893 12.2929L5.58579 7L0.292893 1.70711C-0.0976311 1.31658 -0.0976311 0.683417 0.292893 0.292893Z"
+            fill="white" />
+        </svg>
+      </button >
+      <ul>
+        <li class="menu-link"><a href="#portfolio-section">Portfolio</a></li>
+        <li class="menu-link"><a href="#about-section">About</a></li>
+        <li class="menu-link"><a href="#contact-section">Contact</a></li>
+      </ul>
+    </div>
+  </section> 
```

Comment:
- [ ] it would be better if you stick to the Figma design and try to implement the same background of the menu in order to match design 
- Figma
![image](https://user-images.githubusercontent.com/61790587/173405091-b89698bb-567c-4d87-bbef-9567364ab8a7.png)
- Yours 
![image](https://user-images.githubusercontent.com/61790587/173405125-1323f56e-b93c-4635-836d-3c6dfcb72f07.png)